docs/tutorials/upgrade-extension.md

```diff
@@ -28,42 +28,40 @@ kind: ClusterExtension
 metadata:
   name: argocd
 spec:
```

Comment:
If you open this doc

1 - It should be using `kubectl apply -f - <<EOF` so that we can apply it

![Screenshot 2024-11-11 at 17 55 20](https://github.com/user-attachments/assets/108a1e40-e5f2-4dd8-84a6-23c4d4252a13)

2 - The command should have the value instead of `<extension_name>` ( as you changed in the other places )

kubectl patch clusterextension <extension_name> --type='merge' -p '{"spec": {"source": {"catalog": {"version": "<target_version>"}}}}'

3 - The verification step should either have the name ( why are you changing one place and not the others? )

```
kubectl get clusterextension.olm.operatorframework.io/<extension_name> -o yaml
```










$ kubectl get clusterextension argocd -o yaml
apiVersion: olm.operatorframework.io/v1alpha1
kind: ClusterExtension
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"olm.operatorframework.io/v1alpha1","kind":"ClusterExtension","metadata":{"annotations":{},"name":"argocd"},"spec":{"namespace":"argocd","serviceAccount":{"name":"argocd-installer"},"source":{"catalog":{"packageName":"argocd-operator","version":"0.6.0"},"sourceType":"Catalog"}}}
  creationTimestamp: "2024-11-11T16:50:53Z"
  generation: 4
  name: argocd
  resourceVersion: "42713"
  uid: 4bcaf118-a4cf-4013-842f-501a9da7f597
spec:
  namespace: argocd
  serviceAccount:
    name: argocd-installer
  source:
    catalog:
      packageName: argocd-operator
      upgradeConstraintPolicy: CatalogProvided
      version: 0.6.0
    sourceType: Catalog

Reply:
@trgeiger ^

Reply:
Updated. Note that I did not change number 1 to use `kubectl apply -f - <<EOF`: the snippet you are referring to is a pre-requisite. An example of the original state before upgrade.

Resolving,